applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -458,11 +458,11 @@ validate_user(JObj, Context) ->
                                  ,[{fun cb_context:set_req_data/2, JObj}
                                   ,{fun cb_context:set_req_nouns/2, [{?KZ_ACCOUNTS_DB, []}]}
                                   ,{fun cb_context:set_req_verb/2, ?HTTP_PUT}
-                                  ,{fun cb_context:set_resp_status/2, 'fatal'}
+                                  ,{fun cb_context:set_resp_status/2, success}
```

Comment:
Maybe. Without this validation fails with fatal and no errors though.
I'm trying with the resp_status preset in #cb_context{} now (=error instead of fatal).

I have seen a user validation failure and the reason would be because of missing "first_name" and "last_name" which are however clearly passed in crossbar_maintenance. Lot's of funky things...